.coderabbit.yaml

```
@@ -0,0 +1,245 @@
+    - React Query cache optimization
+    - Bundle size impact and code splitting
+    - Accessibility and semantic HTML
+    - TypeScript strict mode compliance
+    
+  security_patterns: |
+    When reviewing security implementations, validate:
+    - JWT token handling and refresh logic
+    - Input validation and sanitization
+    - CORS and rate limiting configuration
+    - Multi-tenant credential isolation
+    - Encryption for sensitive data storage
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)